patches/minecraft/net/minecraft/gametest/framework/GameTestRegistry.java.patch

```diff
@@ -1,5 +1,56 @@
 --- a/net/minecraft/gametest/framework/GameTestRegistry.java
 +++ b/net/minecraft/gametest/framework/GameTestRegistry.java
+@@ -15,6 +_,8 @@
+ import java.util.function.Function;
+ import java.util.stream.Collectors;
+ import javax.annotation.Nullable;
++
++import net.minecraft.resources.ResourceLocation;
+ import net.minecraft.server.level.ServerLevel;
+ import net.minecraft.world.level.block.Rotation;
+ 
+@@ -25,21 +_,38 @@
+    private static final Map<String, Consumer<ServerLevel>> f_177498_ = Maps.newHashMap();
+    private static final Collection<TestFunction> f_177499_ = Sets.newHashSet();
+ 
++   /** @deprecated Forge: Use {@link net.minecraftforge.event.RegisterGameTestsEvent RegisterGameTestsEvent} to register game tests */
++   @Deprecated
+    public static void m_177501_(Class<?> p_177502_) {
+-      Arrays.stream(p_177502_.getDeclaredMethods()).forEach(GameTestRegistry::m_177503_);
++      register(p_177502_, java.util.Set.of());
++   }
++   /** @deprecated Forge: Use {@link net.minecraftforge.event.RegisterGameTestsEvent RegisterGameTestsEvent} to register game tests */
++   @Deprecated
++   public static void register(Class<?> p_177502_, java.util.Set<String> allowedNamespaces) {
++      Arrays.stream(p_177502_.getDeclaredMethods()).forEach(m -> GameTestRegistry.register(m, allowedNamespaces));
```

Comment:
If the body of this helper gets moved to the event, then this method that is just deprecated does not need to be added via a patch.